Uno.Gallery/Uno.Gallery.Shared/Extensions/XamlDisplayExtensions.cs

```diff
@@ -144,6 +158,18 @@ private static void OnXamlChanged(DependencyObject sender, DependencyProperty dp
 			}
 		}
 
+		private static void OnHideOnMobileChanged(XamlDisplay sender, bool hideOnMobile)
+		{
+			if (hideOnMobile && sender is XamlDisplay)
+			{
+#if __ANDROID__ || __IOS__
+				sender.Visibility = Visibility.Collapsed;
+#else
+				sender.Visibility = Visibility.Visible;
+#endif
```

Comment:
At the end I've the feeling that adding on the `XamlDisplay`
````xaml
android:Visibility="Collapsed"
ios:Visibility="Collapsed"
```
would be enough?

Reply:
@dr1rrb Thanks David. That's absolutely a better way to accomplish this.